bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/DaikinBindingConstants.java

```diff
@@ -24,8 +24,8 @@
  * The {@link DaikinBindingConstants} class defines common constants, which are
  * used across the whole binding.
  *
- * @author Tim Waterhouse <tim@timwaterhouse.com> - Initial contribution
- * @author Paul Smedley <paul@smedley.id.au> - Modifications to support Airbase Controllers
+ * @author Tim Waterhouse tim@timwaterhouse.com - Initial contribution
+ * @author Paul Smedley paul@smedley.id.au - Modifications to support Airbase Controllers
```

Comment:
```suggestion
 * @author <a href="mailto:tim@timwaterhouse.com">Tim Waterhouse</a> - Initial contribution
 * @author <a href="mailto:paul@smedley.id.au">Paul Smedley</a> - Modifications to support Airbase Controllers
```

Would you prefer that I comment all to keep track of them?

Alternative would be to drop the e-mail address entirely, but since it's already there and we are still using the `@author` tag, I would probably favor my proposal above. Ultimately I think we should completely stop using the `@author` tag and leave this to Git. @openhab/add-ons-maintainers - for sharing my opinion. 😉

Reply:
My preference would be to remove the mail addresses. This would match the may we are doing it in other source files. I don't need comments for each occurrence, just give me some time do to it....

Reply:
If I have not overlooked some, I have removed all email addresses from @author tags.
Hope  everybody is fine with that. But it should be fine, credit to the author is still given.